.github/workflows/web.yml

```
@@ -52,8 +52,24 @@ jobs:
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
+    outputs:
+      page_url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     steps:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v4
+  notify-team:
+    needs: deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notifying the team about new report upload
+        uses: slackapi/slack-github-action@v2.1.1
+        with:
+          webhook: ${{ secrets.SLACK_NEW_MAAT_REPORT_WEBHOOK_URL }}
+          webhook-type: webhook-trigger
+          payload: |
+            {
+              "url": "${{ needs.deploy.outputs.page_url }}/#s=${{ env.REPORT_NAME }}",
```

I would also set the pivot


```suggestion
              "url": "${{ needs.deploy.outputs.page_url }}/#s=${{ env.REPORT_NAME }}&p=${{ env.REPORT_NAME }}",
```